client/my-sites/domains/domain-management/list/domain-only.jsx

```diff
@@ -1,38 +1,69 @@
 /**
  * External dependencies
  */
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import React, { PropTypes } from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
 import EmptyContent from 'components/empty-content';
+import QuerySiteDomains from 'components/data/query-site-domains';
 import { domainManagementEdit } from 'my-sites/domains/paths';
+import { getPrimaryDomainBySiteId } from 'state/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
 
-const DomainOnly = ( { domainName, hasNotice, siteId, translate } ) => (
-	<div>
-		<EmptyContent
-			title={ translate( '%(domainName)s is ready when you are.', { args: { domainName } } ) }
-			line={ translate( 'Start a site now to unlock everything WordPress.com can offer.' ) }
-			action={ translate( 'Create Site' ) }
-			actionURL={ `/start/site-selected/?siteSlug=${ encodeURIComponent( domainName ) }&siteId=${ encodeURIComponent( siteId ) }` }
-			secondaryAction={ translate( 'Manage Domain' ) }
-			secondaryActionURL={ domainManagementEdit( domainName, domainName ) }
-			illustration={ '/calypso/images/drake/drake-browser.svg' } />
-		{ hasNotice && (
-			<div className="domain-only-site__settings-notice">
-				{ translate( 'Your domain should start working immediately, but may be unreliable during the first 72 hours.' ) }
+const DomainOnly = ( { primaryDomain, hasNotice, siteId, slug, translate } ) => {
+	if ( ! primaryDomain ) {
+		return (
+			<div>
+				<QuerySiteDomains siteId={ siteId } />
+				<EmptyContent
+					className={ 'domain-only-site__placeholder' }
+					illustration={ '/calypso/images/drake/drake-browser.svg' }
+				/>
 			</div>
-		) }
-	</div>
-);
+		);
+	}
+
+	const domainName = primaryDomain.name;
+
+	return (
+		<div>
+			<EmptyContent
+				title={ translate( '%(domainName)s is ready when you are.', { args: { domainName } } ) }
+				line={ translate( 'Start a site now to unlock everything WordPress.com can offer.' ) }
+				action={ translate( 'Create Site' ) }
```

Comment:
Should we perhaps hide the `Create Site` button if the domain is already conflicting with a Jetpack site? Not sure what would happen if a user created a site using a domain already used by a Jetpack site 😖

Reply:
Nothing. They'd have a site, and the domain would still be pointed to Jetpack.

I don't want to remove it, as we need at least 1 way to get into the `Create a Site` flow, until we decide what other flow for Jetpack we want. :)

Reply:
Right, but they might be confused (I mean - they probably _are_ confused, since they just clicked to create a site with a domain they already have a site). Just as we have customers that try to create a new site by searching for a new domain _on an existing site_, it's totally possible that the user does not understand the concept of a domain-only site. Heck, it's probably quite possible. So I just wanted to remove this possibility of creating even more chaos - WordPress.com site with the same domain as a Jetpack site on the same account? That's a nightmare.
But it's not a blocker for this PR, of course, just something that might want to look into as a follow up.